rpc/gnmi/gnmi-specification.md

```diff
@@ -1400,8 +1402,10 @@ the current data tree on the server. While the path within the subscription
 SHOULD be a valid path within the set of schema modules that the target
 supports, subscribing to any syntactically valid path within such modules MUST
 be allowed. In the case that a particular path does not (yet) exist, the target
-MUST NOT close the RPC, and instead should continue to monitor for the existence
-of the path, and transmit telemetry updates should it exist in the future.
+would send a `sync_response` as the very first message to indicate this, and in
```

Comment:
I still don't like this wording.

I suggest:

```
In the case that a particular path does not (yet) exist, the target should:
 * For `ONCE` subscriptions, return a `sync_response` and close the RPC.
 * For `POLL` subscriptions, return a `sync_response` immediately, and await the next polling request from the client.
 * For `STREAM` subscriptions, if there is no data at a particular path at the start of a subscription, or when a path is sampled, the target should return a `sync_response` immediately. Should the path be created at some later time, the target should begin to transmit updates for the path, according to the specified mode of the subscription.
```

Thoughts?

Reply:
I like it. One question on a previous sentence:

> subscribing to any syntactically valid path within such modules

So, if the path is not schema-compliant, it should return `NOT_FOUND`? i.e. https://pkg.go.dev/google.golang.org/grpc/codes#Code

Should we document this as well?